MeshLib/MeshSearch/ElementSearch.h

```diff
@@ -63,12 +88,23 @@ class ElementSearch final
         }
 
         std::vector<std::size_t> matchedIDs;
-        for (std::size_t i(0); i < pv->getNumberOfTuples(); ++i)
+
+        if (outside_of)
         {
-            if ((*pv)[i] == property_value)
-                matchedIDs.push_back(i);
+            for (std::size_t i(0); i < pv->getNumberOfTuples(); ++i)
+            {
+                if ((*pv)[i] < min_property_value || (*pv)[i] > max_property_value)
+                    matchedIDs.push_back(i);
+            }
+        }
+        else
+        {
+            for (std::size_t i(0); i < pv->getNumberOfTuples(); ++i)
```

Comment:
This is `std::copy_if`

Reply:
Ah, the index ;)
Sorry, then the copy_if is not applicable. It looked so similar to `push_back(*pv[i])`...